ingest/mappings/mineral-observation.json

```diff
@@ -148,6 +142,25 @@
             "type":"string"
           }
         }
+      },
+      "mineral-specimen":{
```

Comment:
Currently it's only recorded in the specimen as "G", "B", ... This could be the next step?